lib/tether/toolbox.go

```diff
@@ -35,8 +36,12 @@ import (
 type Toolbox struct {
 	*toolbox.Service
 
-	config *ExecutorConfig
-	stop   chan struct{}
+	sess struct {
+		sync.Mutex
```

Comment:
That is potentially going to sprinkle a lot more changes into the tether. I would rather put that off for later refactoring. Right now the assumption that I am making is that during Reload() the contents of config do not change.